### PR TITLE
[IMPAC-665] Sync status on dashboard

### PR DESCRIPTION
**Pull request by enizor**
Only displays 'Sync in real time' when there are cube apps running.
When no apps (at all) are running, displays 'No apps are running!'
/!\ Relies on a new Organization method on the hub, relayed by mno-enterprise.
https://github.com/maestrano/maestrano-hub/pull/802
https://github.com/maestrano/mno-enterprise/pull/567

### CODE REVIEW

File: src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee [hunk 1]
```diff
@@ -66,6 +67,8 @@ module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filt
         )
         scope.hasError = scope.connectors && scope.connectors.length > 0 && _.any(scope.connectors, (c) -> isError(c))
 
+        scope.has_running_apps = scope.connectors.length > 0 && scope.has_running_cube
```
Comment (cesar-tonnoir) [inline src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee]: shouldn't it be `|| scope.has_running_cube` ? (= has connectors OR has cubes)

Comment (enizor) [reply src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee]: That should totally be an OR

File: src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filt
       # Variables initialization
       #====================================
       scope.isSyncing = false
-      scope.realtimeSyncing = false
+      scope.has_connector = false
```
Comment (cesar-tonnoir) [inline src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee]: please use `camelCase` in angular projects: `snake_case` is for ruby ;)

File: src/locales/en-AU.json [hunk 3]
```diff
@@ -736,6 +736,7 @@
 
   "impac.components.sync_apps.syncing":                   "Syncing...",
   "impac.components.sync_apps.show_sync_status":          "Show sync status",
+  "impac.components.sync_apps.no_apps_running":           "No apps are running!",
```
Comment (cesar-tonnoir) [inline src/locales/en-AU.json]: I think this is unused

**Pull request merged**